packages/k8s/k3s/build.yaml

```diff
@@ -31,9 +31,11 @@ steps:
   - bash installer.sh agent
   - rm -rf installer.sh
   - chmod +x /usr/bin/k3s
+  - TAG=$INSTALL_K3S_VERSION ./fetch-template.sh
 
 includes:
 - ^/usr/bin/k3s
+- ^config.toml.tmpl
```

Comment:
as a note: this would be installed over /config.toml.tmpl right? maybe we should default to some other dir, e.g. /usr/share/config/kairos/k3s/...

Reply:
yes, good point. I was just playing around with the script. Moved it now.